zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/DecisionDefinitionControllerTest.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import io.camunda.service.DecisionDefinitionServices;
+import io.camunda.service.entities.DecisionDefinitionEntity;
+import io.camunda.service.search.filter.DecisionDefinitionFilter;
+import io.camunda.service.search.query.DecisionDefinitionQuery;
+import io.camunda.service.search.query.SearchQueryResult;
+import io.camunda.service.search.query.SearchQueryResult.Builder;
+import io.camunda.service.search.sort.DecisionDefinitionSort;
+import io.camunda.service.security.auth.Authentication;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+
+@WebMvcTest(DecisionDefinitionController.class)
```

Comment:
❌ We need to enable the Query API for this test.
```suggestion
@WebMvcTest(
    value = DecisionDefinitionQueryController.class,
    properties = "camunda.rest.query.enabled=true"))
```

Reply:
done here https://github.com/camunda/camunda/pull/20535/commits/d08ead22fcaead4f82cab05bfec8495a2f28534d